File: TP 1/java/aque.java
import java.util.Scanner;

public class aque {
    
    static int letrama(String pala) {
        int resp = 0;
        for (int i = 0; i < pala.length(); i++) {
            if (pala.charAt(i) >= 'A' && pala.charAt(i) <= 'Z') {
                resp++;
            }
        }
        return resp;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String pala = "";  
        int resp = 0;

        pala = sc.nextLine();
        while (!pala.equals("FIM")) {
            resp = letrama(pala);
            System.out.println(resp);
            pala = sc.nextLine();
        }

        sc.close(); 
    }
}

File: TP 1/c/aque.c
#include <stdio.h>
#include <string.h>
int letrama(char pala[]){
    int tam=0;
    int resp=0;
    tam=strlen(pala);
    for(int i=0;i<tam;i++){
        if(pala[i]>='A' && pala[i]<='Z'){
        resp=resp+1;
        }
    }
    return (resp);
}

void main(){
    char pala[500]="";
    int resp=0;

    scanf("%[^\n]", pala);
    getchar();
    while(strcmp(pala, "FIM")!=0){
        resp=letrama(pala);
        printf("%d\n", resp);
        scanf(" %[^\n]", pala);
        getchar();

    }



}